src/components/NavigationBar/MainPageNavBarStyled.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import SmallLogo from '@/images/small_logo.svg';
+
+export const NavBarLayout = styled.nav`
+${(props) =>
+    props.type === 'header'
+      ? `
+      position: fixed;
+      z-index: 1;
+      top: ${props.isvisible ? '0' : '-80px'};
+      left: 0;
+        `
+      : `
+          position: abolute;
```

Comment:
Misstyping. + You don't need to make it 'absolute' at all("Check footer styles in dev tools").

Reply:
fixed